pkg/controllers/version.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright © 2022 MicroShift Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package controllers
+
+import (
+	"context"
+	"path/filepath"
+
+	"github.com/openshift/microshift/pkg/assets"
+	"github.com/openshift/microshift/pkg/config"
+	"github.com/openshift/microshift/pkg/version"
+	"k8s.io/klog/v2"
+)
+
+type VersionManager struct {
+	cfg *config.MicroshiftConfig
+}
+
+func NewVersionManager(cfg *config.MicroshiftConfig) *VersionManager {
+	s := &VersionManager{}
+	s.cfg = cfg
+	return s
+}
+
+func (s *VersionManager) Name() string { return "version-manager" }
+func (s *VersionManager) Dependencies() []string {
+	return []string{"kube-apiserver", "openshift-apiserver"}
```

Comment:
```suggestion
	return []string{"kube-apiserver"}
```

Reply:
I believe for config map we only need to wait for the kube-apiserver

Reply:
Fixed in #785